prov/efa/test/efa_unit_test_ep.c

```diff
@@ -310,6 +311,85 @@ void test_efa_rdm_ep_pkt_pool_page_alignment(struct efa_resource **state)
 	fi_close(&ep->fid);
 }
 
+/**
+ * @brief When using LL128 protocol, test the packet allocated from read_copy_pkt_pool
+ *  is 128 byte aligned.
+ * 
+ * @param[in]	state		struct efa_resource that is managed by the framework
+ */
+void test_efa_rdm_read_copy_pkt_pool_128_alignment(struct efa_resource **state)
+{
+	bool iface_initialized;
+	enum fi_hmem_iface iface;
+	struct efa_rdm_pke *pkt_entry;
+	struct efa_rdm_ep *efa_rdm_ep;
+	struct efa_resource *resource = *state;
+
+	/* Override global attribute */
+	iface = FI_HMEM_CUDA;
+	iface_initialized = hmem_ops[iface].initialized;
+	hmem_ops[iface].initialized = true;
+
+	/* rx_readcopy_pkt_pool is only created when application requested FI_HMEM capability */
+	resource->hints = efa_unit_test_alloc_hints(FI_EP_RDM);
+	assert_non_null(resource->hints);
+	resource->hints->caps |= FI_HMEM;
+	resource->hints->domain_attr->mr_mode |= FI_MR_HMEM | FI_MR_ALLOCATED;
+	efa_unit_test_resource_construct_with_hints(
```

Comment:
This involves the logic of HMEM checking during fi_getinfo, which makes this test depend on some other things that beyond the buffer pool creation logic.

You can do it in the following way:
1. Just call efa_unit_test_resource_construct(), which gives you a set of resources (including domain)
2. Modify the mr mode in efa domain to make it has FI_MR_HMEM
3. Create a new efa ep via `fi_endpoint`
4. alloc pke from this new EP.
5. do the check ...